ast/src/analyzed/mod.rs

```diff
@@ -625,21 +628,51 @@ impl PublicDeclaration {
     }
 }
 
+#[derive(Debug, PartialEq, Eq, PartialOrd, Ord, Clone, Serialize, Deserialize, JsonSchema)]
+pub struct SelectedExpressions<Expr> {
+    pub selector: Option<Expr>,
+    pub expressions: Vec<Expr>,
+}
+
+impl<Expr> Default for SelectedExpressions<Expr> {
```

Comment:
Isn't this the derived implementation?

Reply:
Oh, it doesn't work because Expr doesn't need to implement Default, right?

Reply:
It worked, but it was only used in the right SelectedExpression in the Identity impl. I found it more practical to delete this implementation and hardcode the default expression. Thanks!